### DISCUSSION TITLE
Games - Poll

### DISCUSSION THREAD

**Discussion by PSYclownYT** [PSYclown-org/Eliot-s-EPIC-bookmarklets#5] [Polls]
what games should I try to implement?

**Comment by zpenguin360** [comment]
some sort of chatroom cuz u can get eaglercraft and doom on many other sites

**Reply by PSYclownYT** [reply]
Yeah, I can do that

**Reply by zpenguin360** [reply]
thx  th@t w0u1d b3 epik
